app/src/main/kotlin/com/stytch/androidexample/ui/Profile.kt

```diff
@@ -44,7 +44,7 @@ fun Profile(
             }
             (profile.value as? ProfileState.Error)?.let {
                 Text(
-                    text = it.error.reason?.toString() ?: "Unknown error occurred",
+                    text = it.error.toString() ?: "Unknown error occurred",
```

Comment:
Here again, we can use `.message` and since it's guaranteed to exist, we can remove the `?: "Unknown error occurred"`